public_dropin_gpu_environments/nemollm_inference_ms/Dockerfile

```diff
@@ -1,67 +1,4 @@
-# This is the default base image for use with user models and workflows.
-FROM nvcr.io/ohlfw0olaadg/ea-participants/nemollm-inference-ms:24.01
-USER root
-# Need to change the UID of nemo user until upstream fixes this as
-# it is out-of-range for our rootless buildkit image builder.
-RUN groupmod -g 1000 nemo && \
-  usermod -u 1000 -g 1000 nemo && \
-  chown -R 1000.1000 /home/nemo /model-store && \
-  rm /var/log/lastlog
-
-# Need to do this to squash our hack above so rootless buildkit won't
-# try and unpack any layers with out-of-bound UIDs
-FROM scratch
-COPY --from=0 / /
-ARG BASE_PATH=/opt/tritonserver/bin:/usr/local/cmake/bin:/usr/local/cmake/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin:/opt/tritonserver/bin
-
-ENV PATH=${BASE_PATH} \
-    CUDA_VERSION=12.3.2.001 \
-    CUDA_DRIVER_VERSION=545.23.08 \
-    CUDA_CACHE_DISABLE=1 \
-    NVIDIA_REQUIRE_JETPACK_HOST_MOUNTS= \
-    _CUDA_COMPAT_PATH=/usr/local/cuda/compat \
-    ENV=/etc/shinit_v2 \
-    BASH_ENV=/etc/bash.bashrc \
-    SHELL=/bin/bash \
-    NVIDIA_REQUIRE_CUDA=cuda>=9.0 \
-    NCCL_VERSION=2.19.3 \
-    CUBLAS_VERSION=12.3.4.1 \
-    CUFFT_VERSION=11.0.12.1 \
-    CURAND_VERSION=10.3.4.107 \
-    CUSPARSE_VERSION=12.2.0.103 \
-    CUSOLVER_VERSION=11.5.4.101 \
-    CUTENSOR_VERSION=1.7.0.1 \
-    NPP_VERSION=12.2.3.2 \
-    NVJPEG_VERSION=12.3.0.81 \
-    CUDNN_VERSION=8.9.7.29+cuda12.2 \
-    TRT_VERSION=8.6.1.6+cuda12.0.1.011 \
-    TRTOSS_VERSION=23.11 \
-    NSIGHT_SYSTEMS_VERSION=2023.4.1.97 \
-    NSIGHT_COMPUTE_VERSION=2023.3.1.1 \
-    DALI_VERSION=1.32.0 \
-    DALI_BUILD=10610166 \
-    POLYGRAPHY_VERSION=0.49.1 \
-    TRANSFORMER_ENGINE_VERSION=1.1 \
-    LD_LIBRARY_PATH=/usr/local/tensorrt/lib:/usr/local/cuda/compat/lib:/usr/local/nvidia/lib:/usr/local/nvidia/lib64:/opt/tritonserver/backends/tensorrtllm \
-    NVIDIA_VISIBLE_DEVICES=all \
-    NVIDIA_DRIVER_CAPABILITIES=compute,utility,video \
-    NVIDIA_PRODUCT_NAME="Triton Server Base" \
-    GDRCOPY_VERSION=2.3 \
-    HPCX_VERSION=2.16rc4 \
-    MOFED_VERSION=5.4-rdmacore39.0 \
-    OPENUCX_VERSION=1.15.0 \
-    OPENMPI_VERSION=4.1.5rc2 \
-    RDMACORE_VERSION=39.0 \
-    OPAL_PREFIX=/opt/hpcx/ompi \
-    OMPI_MCA_coll_hcoll_enable=0 \
-    LIBRARY_PATH=/usr/local/cuda/lib64/stubs: \
-    NVIDIA_TRITON_SERVER_BASE_VERSION=23.12 \
-    NVIDIA_BUILD_ID=77457706 \
-    DEBIAN_FRONTEND=noninteractive \
-    TRT_ROOT=/usr/local/tensorrt \
-    PYTHONPATH=/opt/nemollm/FasterTransformer:/opt/nemollm/nemo-fw: \
-    NIM_GIT_COMMIT=7a70adeb \
-    NEMO_USER=nemo
+FROM nvcr.io/ohlfw0olaadg/ea-participants/nemollm-inference-ms:dev-24.02
```

Comment:
@elatt it seems to be working :tada:  But we need to pass a correct model name, instead of the `generic_llm` name.  The `nemollm_inference/triton/llm_pipeline/trt_pipeline` tries to get model name on it's own and it is wrong:
```
Internal: FileNotFoundError: [Errno 2] No such file or directory: '/model-store/trt_llm_0.0.1_trtllm/config.pbtxt'
```
here is how it generates a name:
```
/usr/local/lib/python3.10/dist-packages/nemollm_inference/triton/utils/names.py
from importlib.metadata import version


def _get_nim_version():
    return version("nemollm_inference")


def get_full_trtllm_name(model_name):
    return f"{model_name}_{_get_nim_version()}_trtllm"   #  ---> this method is used to get a model name


def get_full_bls_name(model_name):
    return model_name


def get_trtllm_name_from_directory(directory="/model-store/"):
    trt_llm_dirs = glob.glob(f"{directory}/*_trtllm")
    trtllm_model_name = trt_llm_dirs[0].split('/')[-1] if len(trt_llm_dirs) == 1 else "trt_llm"
    return trtllm_model_name

```

Reply:
Ah ok, was this a change in the newer image? Anyway, I see you are handling it transparently for the end user by reading the pbtxt file so that's great 🎉